cli/src/main/kotlin/OrtMain.kt

```diff
@@ -210,6 +215,20 @@ object OrtMain : CliktCommand(name = ORT_NAME, invokeWithoutSubcommand = true) {
     }
 }
 
+/**
+ * Write the [ortResult] to all [outputFiles] for the given [resultName].
+ */
+fun CliktCommand.writeOrtResult(ortResult: OrtResult, outputFiles: Collection<File>, resultName: String) {
+    outputFiles.forEach { file ->
+        println("Writing $resultName result to '$file'.")
```

Comment:
Have you considered moving the `println()` above the for loop, like:
`println("Writing $resultName result to '$outputFiles'.")

The output IMO gives kind of a heads up that this can take a little bit.
So that heads up can be given right away for all files which are planned to be written.

Reply:
No, I haven't considered this (as it was like that before). But what's the advantage of giving this "heads-up" for all files at once? Isn't it even better the way it is now, where the heads-up is given before each file is written, and then by the time of the next heads-up you know that the previous file was fully written? Like a crude progress indicator?